modules/ROOT/pages/about-mule-event.adoc

```diff
@@ -23,8 +23,12 @@ image::about-mule-event-2724e.png[]
 . The Mule event travels sequentially through the components of a flow.
 . Each component interacts in a pre-defined manner with the Mule event.
 
+In some cases when an issue occurs in a Message source, the component does not produce a Mule event. For example, invalid or incorrect paths prevent the creation of a Mule event. The Mule application logs such issues in ERROR, WARN, or INFO messages.
```

Comment:
`The Mule application logs such issues in ERROR, WARN, or INFO messages.` -> This logging is done by the connector, not the app or the runtime.

Reply:
Revised